src/hardware/encoder.cpp

```diff
@@ -1,86 +1,63 @@
 #include "hardware/encoder.h"
+#include "utils/encoderinterrupts.h"
+#include "math.h"
 
-#include <phys253.h>
-#include <avr/interrupt.h>
+namespace hardware {
 
-namespace hardware{
-
-  volatile uint32_t counts[4];
-  volatile unsigned long prevTime[4];
-  const unsigned long WAITTIME = ENCODER_WAIT_TIME();
-
-Encoder::Encoder(){
-  start(R_ENCODER_);
-  start(L_ENCODER_);
+int32_t Encoder::cmToTicks(float cm) {
+  return cm * GEAR_RATIO() * ENCODER_TICKS_PER_REVOLUTION() / (WHEEL_DIAMETER() * M_PI);
```

Comment:
lets have this as a constant, so we don't have to do the calculation again and again.